footer.php

```diff
@@ -2,29 +2,27 @@
 		</div><!-- container footer-widgets -->
 </div><!-- wrapper container -->
 
-		<div class="container" id="foot">
+		<footer class="container" id="foot">
 			<div class="row">
 
-				<div class="span3">
+				<nav class="span3">
                 	
                      <div class="vcard">
                          <div class="org"><h4><a class="url fn n" href="<?php echo home_url(); ?>/">KBCS Radio</a></h4></div>
                          
-                         <div class="adr">
-                              <div class="street-address">3000 Landerholm Circle SE</div>
-                              <span class="locality">Bellevue</span>,
-                              <span class="region">WA</span>
-                              <span class="postal-code">98007-6406</span>
-                         </div>
+                         <address class="adr">
+                              <p class="street-address">3000 Landerholm Circle SE</p>
+							  <p class="street-address">Bellevue, WA 98007-6406</p>
+                         </address>
                      
                      	<ul>    
                             <li><a href="<?php echo home_url(); ?>/about/contact/"><strong>Contact Us</strong></a></li>
                             <li><a href="<?php echo home_url(); ?>/about/directions/"><strong>Map &amp; directions</strong></a></li>
                         </ul>
                    </div><!-- .vcard -->
-				</div><!-- span3 -->
+				</nav><!-- span3 -->
                 
-                <div class="span3">
+                <nav class="span3">
```

Comment:
I would recommend placing all of these in a single `<nav>` instead of having multiple small `<nav>` elements in the footer